doc/userguide/src/Appendices/DocumentationFormatting.rst

```diff
@@ -22,70 +22,14 @@ __ `Documenting libraries`_
 Representing newlines
 ---------------------
 
-Adding newlines manually in test data
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The documentation of test suites, test cases and keywords as well as
-the test suite metadata are subject to `general parsing rules`__ of the
-test data. This means that normal newlines are not preserved and
-dividing documentation into lines and paragraphs generally requires
-using a `literal newline character sequence`__ (`\n`). This is
-shown in the example below.
-
-__ `Test data syntax`_
-__ `Handling whitespace`_
-
-.. raw:: html
-
-   <table class="example docutils">
-     <caption>Adding newlines manually</caption>
-     <tr>
-       <th>Setting</th>
-       <th>Value</th>
-       <th>Value</th>
-     </tr>
-     <tr>
-       <td>Documentation</td>
-       <td>
-         First line.\n<br>\n<br>
-         Second paragraph, this time\n<br>with multiple lines.
-       </td>
-       <td>&nbsp;</td>
-     </tr>
-     <tr>
-       <td>Metadata</td>
-       <td>Example</td>
-       <td>Value\n<br>in two lines</td>
-     </tr>
-   </table>
-
 Automatic newlines in test data
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This title is now misleading because the section below talks both about adding newlines manually and how they are automatically added.

I like the new section text, though, and understand documentation here should be cleaned up. As I already commented above, I'd like this PR to contain only example format changes.